spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/DefaultTaskExecutionService.java

```diff
@@ -196,23 +196,28 @@ public long executeTask(String taskName, Map<String, String> taskDeploymentPrope
 
 		if (taskExecutionInformation.isComposed()) {
 			boolean containsAccessToken = false;
+			boolean useUserAccessToken = false;
 
-			final String dataflowAccessTokenKey = "dataflow-server-access-token";
+			final String dataflowServerAccessTokenKey = "dataflow-server-access-token";
+			final String dataflowServerUseUserAccessToken = "dataflow-server-use-user-access-token";
```

Comment:
This new key should somewhat get documented somewhere in `spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc`.

Reply:
Added docs.